com.unity.multiplayer.mlapi/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -307,7 +314,7 @@ private static void OnSceneUnloadServer(Guid switchSceneGuid)
                                 }
                                 else
                                 {
-                                    writer.WriteUInt32Packed(newSceneObjects[i].GlobalObjectIdHash);
+                                    writer.WriteBool(false);
```

Comment:
what's this? (also `writer.WriteBool(true);` above) would you like to comment next to them to make life easier for future maintainers? :)

Reply:
Updated that whole area with comments to better clarify what is happening.